README.md

```diff
@@ -277,3 +188,25 @@ fun (rw : Reader & Writer) {
     result
 }
 ```
+
+Read more about capabilities [here](doc/03_capabilities.md).
+
+## Participants
+
+* Anthony Sloane (Anthony.Sloane@mq.edu.au)
```

Comment:
Please change my email to `inkytonik@gmail.com` since eventually my MQ email will go away.